src/liquid/components/ld-bg-cells/readme.md

```diff
@@ -0,0 +1,282 @@
+---
+eleventyNavigation:
+  key: Background Cells
+  parent: Components
+layout: layout.njk
+title: Background Cells
+permalink: components/ld-bg-cells/
+---
+
+# ld-bg-cells
+
+A background pattern with the Merck cells as additional visual element.
+
+---
+
+## Default
+
+{% example %}
+<ld-bg-cells class="box1-16/9"></ld-bg-cells>
+
+<style>
+  .box1-16\/9 {
+    margin: 0 0 calc(var(--ld-sp-24) * -1 + 1px) !important;
+    padding-top: 56.25%;
+  }
+</style>
+{% endexample %}
+
+<!-- CSS component -->
+
+<!-- <div class="ld-bg-cells ld-bg-cells--safc box1-16/9">
+  <div class="ld-bg-cells__pattern"></div>
+</div>
+
+<style>
+  .box1-16\/9 {
+    margin: 0 0 calc(var(--ld-sp-24) * -1 + 1px) !important;
```

Comment:
I think we don't need to set a margin here at all, because
1. It does not remove the white space around the component completely (because there is still some padding and border visible, especially in dark mode)
2. It may confuse the reader of the documentation
3. Could be taken care of differently: namely be adding another prop to the docs-example component which makes sure no default margins and paddings are set (not worth the effort in this case; I think the white space around the component is ok)
4. `!important` 🙈